packages/style-loader/index.js

```diff
@@ -8,16 +8,12 @@ module.exports = (neutrino, opts = {}) => {
   const options = merge({
     test: neutrino.regexFromExtensions(['css']),
     ruleId: 'style',
-    styleUseId: 'style',
-    cssUseId: 'css',
     css: {
       importLoaders: opts.loaders ? opts.loaders.length : 0
     },
     style: {},
     modules,
     modulesTest,
-    modulesSuffix: '-modules',
-    exclude: modules && modulesTest,
     loaders: [],
     extractId: 'extract',
```

Comment:
> I left `extractId` as we seem to do that with other added plugins, and I could see that being a conflict someone had to override.

I think the main use-case for overriding `extractId` might be someone wanting to use `@neutrinojs/style-loader` twice and have mini-css-extract-plugin be added to the configuration twice under different Ids. However mini-css-extract-plugin isn't designed to be used more than once, and doesn't provide a way to filter the set of files on which it operates (ie so not possible to use plugin instance for css modules vs normal css).

As such, should we drop it too? We could always re-add if someone requests it -- it just seems like throughout all of the presets we've perhaps pre-emptively added configuration options before knowing of use-cases which actually require them.

Reply:
> it just seems like throughout all of the presets we've perhaps pre-emptively added configuration options before knowing of use-cases which actually require them

I think that is true. Realistically, if any mw would need to provide this level of customization, it would be a third-party one, not a core one like this.

I'll get rid of it.

Do you think its worth auditing other middlewares for similar options to drop?

Reply:
> Do you think its worth auditing other middlewares for similar options to drop?

Yeah quite possibly (in other PRs) :-)

Reply:
All set.